kubewarden-load-policies.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+policies="kubewarden-policies.tar.gz"
+list="kubewarden-policies.txt"
+
+usage () {
+    echo "USAGE: $0 [--policies kubewarden-policies.tar.gz] --registry my.registry.com:5000"
+    echo "  [-l|--policies-list path] text file with list of policies; one image per line."
+    echo "  [-p|--policies path] tar.gz generated by kwctl save."
+    echo "  [-r|--registry registry:port] target private registry:port."
+    echo "  [-s|--sources-path path] kwctl sources path."
+    echo "  [-h|--help] Usage message"
+}
+
+pushPolicy() {
+    newPolicyUrl=$1
+    if [[ -n $sourcesPath ]]; then
+        kwctl push $policy $newPolicyUrl --sources-path $sourcesPath
+    else
+        kwctl push $policy $newPolicyUrl
+    fi
+}
+
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        -r|--registry)
+        registry="$2"
+        shift # past argument
```

Comment:
You can move one `shift` after `key="$1"`, out of case block to not repeat it for every argument.
Or use `shift 2` if you are shifting twice.

Reply:
not sure if that doesn't work, or if I'm missing something (I'm not an expert in bash :sweat_smile: ) I took this code from the `rancher-save-images.sh` script which uses the same switch.

I get the following error:
`./scripts/kubewarden-load-policies.sh: line 43: $2: unbound variable`
I think this is because shift needs to happens after `policies="$2"` and not before as it happens if I move the shift out of the switch block after key="$1".

Reply:
done [here](https://github.com/kubewarden/kwctl/pull/354/commits/20b0c57ea52b1df5d46cd7cf661ccae8d4969c77). I needed to change `policies="$2"` to `policies="$1"` as you suggested. Thanks